ChegeBryan/politico
# #163683369 Setup development tools

#### What does this PR do?

Setup the development tools to be used.

#### Description of Task to be completed?

- [x] Write tests for environment configurations.
- [x] Write the app configurations.

#### How should this be manually tested?
After cloning the repository to your local machine - check
[README.md](README.md) for instructions. Checkout to `ch-setup-development-tools-163683369` branch and launch the flask server.  

#### What are the relevant pivotal tracker stories?
[#163683369](https://www.pivotaltracker.com/story/show/163683369)
